### PR TITLE
Adding definitions of Golden tags and golden metrics

### PR DESCRIPTION
**Pull request by AlbertoGoYu**
### Updated Golden Tags and Golden Metrics

Edited golden tags and golden metrics for following integrations: 

- AWSALB
- AWSALBTARGETGROUP
- AWSLAMBDAFUNCTION
- AWSLAMBDAREGION
- AWSNLB
- AWSNLBTARGETGROUP
- AWSREDSHIFTCLUSTER
- AWSREDSHIFTNODE

### CODE REVIEW

**Review by cianBuckley — APPROVED** [review]

**Review by naxhh — COMMENTED** [review]
@AlbertoGoYu thanks for this one!

Seems there are a bunch of conflicts to resolve before we can merge this one

File: definitions/infra-awsalb/golden_metrics.yml [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
   query:
-    select: sum(provider.requestCount.Sum)
+    select: rate(sum((provider.httpCodeElb4XXCount.Sum OR 0) + (provider.httpCodeElb5XXCount.Sum OR 0)),1 minute)
     from: LoadBalancerSample
     where: provider='Alb'
     facet: entityName
     eventId: entityGuid
-serverErrors4XxAnd5Xx:
-  title: Server errors (4xx and 5xx)
+requests:
```
[the remark this answers is not in the document]
Comment (AlbertoGoYu) [reply definitions/infra-awsalb/golden_metrics.yml:9]: You are right it's active connections

**Review by AlbertoGoYu — COMMENTED** [review]
Corrected based on feedback

**Comment by AlbertoGoYu** [comment]
Edited to resolve conflicts and comments

**Review by IreneMLC — APPROVED** [review]

**Review by AlbertoGoYu — COMMENTED** [review]
All checked and waiting for approval

**Review by naxhh — APPROVED** [review]
LGTM! I'll spend some time deploying them tomorrow.

Thanks for contributing them!

**Pull request merged**